### DISCUSSION TITLE
[Esame 17 aprile 2024] Esercizio 4

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#461] [Esami M-Z]
![17 apr es.4](https://github.com/user-attachments/assets/a5f40eb1-c9d1-4805-9116-590552d81baf)

**Comment by AglaiaNorza** [comment]
Soluzione ad opera della tutor [Elisa Scandiuzzi](https://github.com/sapienzastudentsnetwork/algebra/blob/main/scandiuzzi/README.md), lezione del 29 ottobre 2024 (`appunti_24_10_29.pdf`):

![1](https://github.com/user-attachments/assets/8be9aff6-b1ce-4f17-af81-fe1db4dec9d2)
![2](https://github.com/user-attachments/assets/1cbc2e9e-3fc4-414f-aa97-c55e6c590ee1)

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/3ef41607-47fd-4ce5-a8da-955df92476b2)
